### DISCUSSION TITLE
Idea about 'Text Color' option

### DISCUSSION THREAD

**Discussion by ostap667inbox** [Nevysha/Cozy-Nest#93] [Ideas]
I enjoy using Cozy Nest, I'm happy with the styles and settings in it. However, I have already heard from several people that they refused to use it because it 'hurts their eyes'. After some clarification I found out that they do not like the high contrast between the white text and the dark background in most of the lettering. This, of course, is an individual peculiarity of vision.

Instead they use other styles where the contrast between the background and the color of the text is less. Most likely, if they had an option to change the default white text color to a darker one, such users would be more comfortable with the Cozy Nest interface.

![add_text_color](https://github.com/Nevysha/Cozy-Nest/assets/123593542/d255f532-63e6-4cf4-b595-4f88566ae6d9)

**Comment by ostap667inbox** [comment]
Yes, there is _more than one variable in the styles_ that is responsible for the color of the text. 
![text_colors](https://github.com/Nevysha/Cozy-Nest/assets/123593542/40662cac-e1ef-495b-9184-ed140d507a4f)

- As an option, it is possible to set the base color of the text, and the rest colors do a gradation of its transparency. The method has both pluses and minuses.

- **Or, alternatively, instead of selecting a text color, make _several color schemes_ with different contrasts.** In this way, by the way, you can also switch between dark and light themes, without changing the address bar of the browser.

**Comment by Nevysha** [comment]
This should be doable to rationalize all those color for text as I've done this for the light theme. I "just" have to refactor the css variable and add a setting for it I think.

**Reply by ostap667inbox** [reply]
> This should be doable to rationalize all those color for text as I've done this for the light theme. I "just" have to refactor the css variable and add a setting for it I think.

What do you think of the drop-down menu with some preset color schemes in the settings? Add the current dark and light schemes there, as well as a couple of options with slightly less contrasty colors for those with heightened light sensitivity.

Seems like a simultaneous solution to a number of things at once:
- Users will get more flexible color scheme customization,
- no need to manually change the URL in the address bar for the light/dark theme if the color scheme is to be kept with the other settings.

**Comment by Nevysha** [comment]
I think that too many option is overkill (and to be able to change things such as bg color, you have to enable to change like, every css var)
